proto/cosmos/tx/v1beta1/tx.proto

```diff
@@ -221,3 +221,24 @@ message Tip {
   // tipper is the address of the account paying for the tip
   string tipper = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
 }
+
+// AuxSignerData is the intermediary format that an auxiliary signer (e.g. a
+// tipper) builds and sends to the fee payer (who will build and broadcast the
+// actual tx). AuxSignerData is not a valid tx in itself, and will be rejected
+// by the node if sent directly as-is.
+//
+// Since: cosmos-sdk 0.45
+message AuxSignerData {
+  // address is the bech32-encoded address of the auxiliary signer. If using
+  // AuxSignerData across different chains, the bech32 prefix of the target
+  // chain (where the final transaction is broadcasted) should be used.
+  string address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+	// sign_doc is the SIGN_MOD_DIRECT_AUX sign doc that the auxiliary signer
+  // signs. Note: we use the same sign doc even if we're signing with
+  // LEGACY_AMINO_JSON.
+	SignDocDirectAux sign_doc = 2;
+  // mode is the signing mode of the single signer
+  cosmos.tx.signing.v1beta1.SignMode mode = 3;
```

Comment:
One thing I thought about looking at this: what if the aux signer is a legacy multisig? Do we want to support that?

If yes we should probably do:
```suggestion
  cosmos.tx.signing.v1beta1.ModeInfo mode_info = 3;
```

Reply:
Say we have a 2/2 multisig as tipper. The 2 signers will use AuxTxBuilder, and use a single mode info anyways.

Who should aggregate the signatures? Probably the fee payer I'd say? In which case I believe it's actually better API to keep `cosmos.tx.signing.v1beta1.SignMode mode = 3;` here, and maybe add a `AddMultisigAuxSignerData` on the fee payer's txbuilder